packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import { useRouter } from "next/navigation";
+
+import { Button } from "react-aria-components";
+
+import { WalkthroughInfo } from "../../../data/src/useWalkthroughData";
+
+import "./WalkthroughCard.css";
+
+export interface WalkthroughCardProps {
+  id: string;
+  data: WalkthroughInfo;
+  href?: string;
+}
+
+export default function WalkthroughCard({
+  id,
+  data,
+  href,
+}: WalkthroughCardProps) {
+  const router = useRouter();
+
+  const handleClick = () => {
+    if (href) router.push(href);
+  };
+
+  const canDisplayCard =
```

Comment:
I don't think we need this check based on all the types and properties being required. We'd get an error in useWalkthroughData.ts if any imported JSON didn't have the appropriate WalkthroughInfo.

Reply:
True

Reply:
Any reason I'm missing why we still have it then?